### DISCUSSION TITLE
Not all remix icons working?

### DISCUSSION THREAD

**Discussion by matthiasautrata** [writer/writer-framework#79] [Q&A] [answered]
On a Button, "arrow-right" works and "arrow-right-double" does not. Are you supporting a certain subset of these icons only?

**Comment by ramedina86** [comment, marked as answer]
Thanks a lot for reporting this, several icons were recently added to RemixIcon and the version used by Streamsync was slightly outdated. I've corrected this and it'll be available in the next release.
